src/MultiOwnable.sol

```diff
@@ -96,14 +94,14 @@ contract MultiOwnable {
         _;
     }
 
-    /// @notice Convenience function to add a new owner address.
+    /// @notice Add a new Ethereum address owner.
     ///
     /// @param owner The owner address.
     function addOwnerAddress(address owner) external virtual onlyOwner {
         _addOwnerAtIndex(abi.encode(owner), _getMultiOwnableStorage().nextOwnerIndex++);
     }
 
-    /// @notice Convenience function to add a new owner passkey.
+    /// @notice Adds a new public key owner.
```

Comment:
Nit but should we conjugate? See `addOwnerAddress()` where we state "Add a new Ethereum address owner."